pkg/adaptor/hypervisor/vsphere/service.go

```diff
@@ -44,7 +44,7 @@ type hypervisorService struct {
 }
 
 func newService(govmomiClient *govmomi.Client, config *Config, hypervisorConfig *hypervisor.Config, workerNode podnetwork.WorkerNode, podsDir, daemonPort string) pb.HypervisorService {
-	logger.Printf("service config %v", config)
+
```

Comment:
How about using `logger.Printf("service config %v", config.Redact())` ?

Reply:
It is an unnecessary duplicate log statement. That info is already printed out in  NewServer.

Reply:
ok